scanner/src/main/kotlin/scanners/scanoss/ScanOssConfig.kt

```diff
@@ -31,7 +31,10 @@ internal data class ScanOssConfig(
     val apiUrl: String,
 
     /** API Key required to authenticate with the ScanOSS server. */
-    val apiKey: String
+    val apiKey: String,
+
+    /** A list of extensions for files (without the dot) to be ignored during the scan. */
+    val ignoredFileExtensions: List<String>
```

Comment:
This should default to the list from `reference.yml`, otherwise every user has to set it now.

Reply:
This actually shows that we seem to be missing a test for ignored file extensions.

Reply:
> This should default to the list from `reference.yml`, otherwise every user has to set it now.

Thanks for noticing. I've used the opportunity to also rework the code to work with suffixes instead of extensions, like in the maintain [SCANOSS Python client](https://github.com/scanoss/scanoss.py/blob/main/src/scanoss/scanner.py#L54-L75).